app/DataTables/SensorDataDataTable.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\DataTables;
+
+use Yajra\DataTables\Services\DataTable;
+use App\SensorData;
+
+class SensorDataDataTable extends DataTable
+{
+    /**
+     * Build DataTable class.
+     *
+     * @return \Yajra\DataTables\Engines\BaseEngine
+     */
+    public function dataTable($query)
+    {
+        return datatables($query)
```

Comment:
The method editColumn does only exist in Yajra\DataTables\DataTableAbstract, but not in Yajra\DataTables\DataTables.